modmesh/app/euler1d.py

```diff
@@ -34,10 +34,15 @@
 import numpy as np
 
 import matplotlib
-from matplotlib.backends.qt_compat import QtWidgets
+import matplotlib.pyplot
 from matplotlib.backends.backend_qtagg import (
-    FigureCanvas, NavigationToolbar2QT as NavigationToolbar)
+    FigureCanvas)
```

Comment:
This doesn't need a new line.  Please keep it in the same line as above.

Reply:
Fix it in https://github.com/solvcon/modmesh/pull/210/commits/214b4743bae1995cf7c451855880a397eae559fc